src/main/java/com/archisacadeny/student/StudentService.java

```diff
@@ -18,6 +18,10 @@ public Student createStudent(Student student){
         return studentRepository.createStudent(student);
     }
 
+    public void deleteStudent(int studentId) {
+        studentRepository.deleteStudent(studentId);
```

Comment:
why there is not logging and printing info about the status of deletion?

Reply:
It is located in the method in studentrepository. That's why I didn't do it there. I didn't know how to move it here. When I typed there again, the message came twice.